symfony/src/Service/EnvironmentService.php

```diff
@@ -171,4 +171,10 @@ protected function loadConfiguration($override = [])
             $this->configuration[$key] = $val;
         }
     }
+
+    public function getSessionTimeout()
+    {
+        return $this->isLocal() ? 3600 * 24 : 30 * 60;
```

Comment:
Isn't this already defined in the constructor on line 49?

Reply:
Yeah you are right, I'll fix it. Thanks!